### DISCUSSION TITLE
Contractions are being highlighted as typos

### DISCUSSION THREAD

**Discussion by paddyroddy** [crate-ci/typos#1094] [Q&A]
This seems to be a recent regression (as well as more false positives). This is putting me off using `typos`, have used it successfully for over a year, but it's now failing on a lot of my repos incorrectly.
```yaml
error: `ve` should be `voe`
  --> renovate-automating-dependency-management/index.qmd:19:7
   |
19 | ## We've All Done This At Some Point
   |       ^^
   |
error: `ve` should be `voe`
  --> README.md:6:21
  |
6 | A series of talks I've given
  |                     ^^
  |
error: `ve` should be `voe`
  --> pyopensci-my-experience/index.qmd:39:59
   |
39 | > We built this journal because we believe that after you've done the hard work
   |                                                           ^^
   |
```

**Comment by epage** [comment]
Could you give reproduction steps?  I'm not able to reproduce this just by copying one of those lines and running typos on it.

**Reply by paddyroddy** [reply]
I'm running via `pre-commit`, with the following `.pre-commit-config.yaml`
```yaml
---
repos:
  - repo: https://github.com/crate-ci/typos
    rev: v1.24.3
    hooks:
      - id: typos
        args:
          - --force-exclude
          - --hidden
          - --locale=en-gb
```

**Reply by paddyroddy** [reply]
Just tried again with `v1.24.4` (released in the last 11 hours) and this is no longer happening. So I guess this can be closed?
